jekyll/_cci2/workflows.adoc

```diff
@@ -34,35 +34,45 @@ For example, if only one job in a workflow fails, you will know it is failing in
 Workflows may appear with one of the following states:
 
 [.table.table-striped]
-[cols=2*, options="header", stripes=even]
+[cols=3*, options="header", stripes=even]
 |===
-| State | Description
+| State | Description | Terminal State
 
 | RUNNING
 | Workflow is in progress
+| No
 
 | NOT RUN
 | Workflow was never started
+| No
 
 | CANCELED
 | Workflow was canceled before it finished
+| Yes
 
 | FAILING
 | A job in the workflow has failed
+| No
 
 | FAILED
 | One or more jobs in the workflow failed
+| Yes
 
 | SUCCESS
 | All jobs in the workflow completed successfully
+| Yes
 
 | NEEDS APPROVAL
 | A job in the workflow is waiting for approval
+| No
 
 | NEEDS SETUP
 | A workflow stanza is not included or is incorrect in the xref:configuration-reference#[`.circleci/config.yml` file] for this project
+| No
 |===
```

Comment:
`NEEDS SETUP` is not actually a workflow state we support. From the description I think what is meant here is `ERROR`, which is a terminal status. I've searched the UI to verify, and it does not look like this is a case of the UI renaming the `ERROR` state to `NEEDS SETUP` (like it does for `ON HOLD` -> `NEEDS APPROVAL`)

```suggestion
| ERROR
| A workflow stanza is not included or is incorrect in the xref:configuration-reference#[`.circleci/config.yml` file] for this project
| Yes

|
| UNAUTHORIZED
| One or more of the jobs terminated with a `unauthorized` job status, because the triggerer does not have access to a required restricted context.
| Yes
|===
```